Config/Filter/Type/Background.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of the `liip/LiipImagineBundle` project.
+ *
+ * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Liip\ImagineBundle\Config\Filter\Type;
+
+use Liip\ImagineBundle\Config\Filter\Argument\Size;
+use Liip\ImagineBundle\Config\FilterInterface;
+
+/**
+ * @codeCoverageIgnore
+ */
+final class Background extends FilterAbstract implements FilterInterface
```

Comment:
you can remove the `implements FilterInterface` because thats already given by extending `FilterAbstract`

Reply:
done